### DISCUSSION TITLE
What is your favorite food? あなたの一番好きな食べ物を教えてください。

### DISCUSSION THREAD

**Discussion by ShizukuIida** [NAIST-SE/AttractingContributors#534] [General]
Please tell me about your favorite food.
あなたの一番好きな食べ物を教えてください。

**Example**
1. Steak
2. Takoyaki
3. ...

The repository is here.
https://github.com/ShizukuIida/AttractingContributors-Food

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 9th food in your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
